tests/yast2_gui/yast2_software_management.pm

```diff
@@ -9,16 +9,124 @@
 # without any warranty.
 
 # Summary: Test YaST2 module for software management
-# Maintainer: Max Lin <mlin@suse.com>
+# Maintainer: Zaoliang Luo <zluo@suse.de>
 
 use base "y2x11test";
 use strict;
 use testapi;
+use version_utils qw(is_sle is_leap is_tumbleweed);
+
+sub sw_single {
+
+    # check menu entries at first, then continue with View, Search, RPM Groups
+    # and Installation Summary
+
+    # File - open Load package List
+    assert_and_click 'sw_file_open';
+    assert_and_click 'sw_file_import';
+    assert_and_click 'package-load-package-list_cancel';
+    wait_still_screen 60;
+
+    # Package - All packages - Update
+    assert_and_click 'sw_package';
+    # use mouse_set and click All in the List because assert_and_click won't hold
+    # the menu open for update if newer version available
+    mouse_set(184, 354);
```

Comment:
This won't scale, we should never use coordinates. What is exactly the problem here? Should not be too hard to get it in the os-autoinst

Reply:
assert_and_click won't keep menu open, so this is workaround. please let me know if you have a better solution for this.